filesystem/iso9660.ksy

```diff
@@ -7,206 +7,84 @@ meta:
     pronom: fmt/468
     wikidata: Q815645
   license: CC0-1.0
+  endian: be
```

Comment:
```suggestion
  ks-version: 0.9
  bit-endian: be
  endian: be
```

The spec uses [big-endian bit-sized integers](https://doc.kaitai.io/user_guide.html#bit-ints-be) and it's better to make this explicit with the [`bit-endian` key](https://doc.kaitai.io/user_guide.html#bit-endian). Also `bit-endian` and `valid` were added in [0.9](https://kaitai.io/news/2020/10/16/kaitai-struct-v0.9-released.html), so let's set that as a minimum required KS version (although not really a big deal, because only a few languages' codegen actually cares about the `ks-version` key and also with every major release we make become all previous versions EOL).

Reply:
fixed